source/virtualBuffers/gecko_ia2.py

```diff
@@ -23,9 +23,24 @@
 class Gecko_ia2_TextInfo(VirtualBufferTextInfo):
 
 	def _normalizeControlField(self,attrs):
+		#log.info("normalizeControlField attrs: %s", attrs)
 		ariaCurrent = attrs.get("IAccessible2::attribute_current")
 		if ariaCurrent != None:
 			attrs['current']= ariaCurrent
+		ariaPlaceholder = attrs.get("IAccessible2::attribute_placeholder")
+		# For efficiency, only check if it is valid to include placeholder when we have a placeholder value to include.
+		if ariaPlaceholder != None:

```

Comment:
Change this to either `if placeholder` or (if you're worried about the difference between None and empty string) `if placeholder is not None`.